tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.jsx

```diff
@@ -44,7 +44,7 @@ const SubmissionHistoryRow = ({ file }) => {
       <td>
         {file.hasError > 0 ? (
           <button className="section-download" onClick={returned_errors}>
-            {file.year}-{file.quarter}-{file.section}
+            {file.year}-{file.quarter}-{file.section}.xlsx
```

Comment:
From a purity standpoint, this shouldn't be here and should be a one-off PR but I am not willing to die on that hill. Why was it included though?

Reply:
The team discussed including it in the 1610 scope since i was already going to work on some frontend column updates, i was asked to put it in the part 1 pr instead of part 2 just to get it through faster. I don't mind moving it to part 2 or creating a separate pr, though.